mdx/api/index.mdx

```diff
@@ -58,6 +58,30 @@ And will only document the objects in the `records` field.
 If a query provides an unreasonable (ie. too high) `perPage` value, the response can return a validation error as specified in the [errors section](#errors).
 If the query specifies a `page` that does not exist (ie. there are not enough `records`), the response will return an empty `records` array.
 
+
+## Allowance Targets
+Some interactions with 0x require or are improved by setting [token allowances](https://tokenallowance.io/), or in other words, giving 0x's smart contracts permission to move certain tokens on your behalf. Some examples include - 
+
+* Submitting a 0x API quote selling ERC20 tokens, you will need to give an allowance to the contract address. This address can be found either from the `allowanceTarget` quote response or from the ExchangeProxcy Address from (#addresses by network) table below. 
```

Comment:
> This address can be found either from the `allowanceTarget` quote response or from the ExchangeProxcy Address from (#addresses by network) table below. 

Maybe change this to: This address can be found either as the value of `allowanceTarget` returned in the quote response or in the ExchangeProxy Address column in the (#addresses by network) table below.